library/core/src/ptr/non_null.rs

```diff
@@ -1,3 +1,4 @@
+use super::*;
```

Comment:
Is this needed?

Reply:
Without this, the program cannot compile due to thousands of `error[E0034]: multiple applicable items in scope`

Reply:
Can you post one of those errors? Generally, we should avoid making changes that impact the non-verification (i.e. normal build) path.

Reply:
```
error[E0773]: attempted to define built-in macro more than once
    --> /rustc/bd53aa3bf7a24a70d763182303bd75e5fc51a9af/library/core/src/macros/mod.rs:1468:5
     |
note: previously defined here
    --> core/src/macros/mod.rs:1468:5
     |
1468 | /     macro_rules! cfg {
1469 | |         ($($cfg:tt)*) => {
1470 | |             /* compiler built-in */
1471 | |         };
1472 | |     }
     | |_____^

error[E0425]: cannot find function `null_mut` in this scope
    --> core/src/ptr/non_null.rs:1805:44
     |
1805 |         let null_ptr = NonNull::<u32>::new(null_mut());
     |                                            ^^^^^^^^ not found in this scope
     |
help: consider importing one of these functions
     |
1786 +     use crate::ptr::null_mut;
     |
1786 +     use core::ptr::null_mut;
     |

error[E0034]: multiple applicable items in scope
    --> core/src/iter/adapters/take.rs:282:22
     |
282  |                 more.checked_sub(1)
     |                      ^^^^^^^^^^^ multiple `checked_sub` found
     |
     = note: candidate #1 is defined in an impl for the type `usize`
note: candidate #2 is defined in an impl for the type `usize`
    --> core/src/num/uint_macros.rs:655:9
     |
655  |           pub const fn checked_sub(self, rhs: Self) -> Option<Self> {
     |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
    ::: core/src/num/mod.rs:1260:5
     |
1260 | /     uint_impl! {
1261 | |         Self = usize,
1262 | |         ActualT = u64,
1263 | |         SignedT = isize,
...    |
1277 | |         bound_condition = " on 64-bit targets",
1278 | |     }
     | |_____- in this macro invocation
     = note: this error originates in the macro `uint_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0034]: multiple applicable items in scope
   --> core/src/future/async_drop.rs:204:46
    |
204 |         unsafe { async_drop_in_place_raw(ptr.add(i)).await }
    |                                              ^^^ multiple `add` found
    |
    = note: candidate #1 is defined in an impl for the type `*mut T`
note: candidate #2 is defined in an impl for the type `*mut T`
   --> core/src/ptr/mut_ptr.rs:950:5
    |
950 | /     pub const unsafe fn add(self, count: usize) -> Self
951 | |     where
952 | |         T: Sized,
    | |_________________^
```
`error: could not compile `core` (lib) due to 4515 previous errors`

Reply:
I was able to reproduce this. The number of errors emitted is unbelievable!

Anyhow, you can remove the `use super::*`, and instead add the following inside the `verify` module:
```rust
use crate::ptr::null_mut;
```
This should remove all the errors.